xml/decoders/lenz/protection_cv30.xml

```diff
@@ -92,17 +104,20 @@
         <choice xml:lang="it">OK</choice>
         <choice xml:lang="de">Okay</choice>
         <choice xml:lang="cs">V pořádku</choice>
+        <choice xml:lang="ca">Correcte</choice>
       </enumChoice>
       <enumChoice choice="Fault">
         <choice>Fault</choice>
         <choice xml:lang="it">Guasto</choice>
         <choice xml:lang="de">Fehler</choice>
         <choice xml:lang="cs">Závada</choice>
+        <choice xml:lang="ca">Error</choice>
       </enumChoice>
     </enumVal>
     <label>Motor short-circuit</label>
     <label xml:lang="it">Corto circuito Motore</label>
     <label xml:lang="de">Motor Kurzschluss</label>
     <label xml:lang="cs">Zkrat motoru</label>
+    <choice xml:lang="ca">Cort-circuit al motor</choice>
```

Comment:
This should probably be a `<label>` element, not a `<choice>` element.  The `<choice>` here is failing validation.